actioncable/README.md

```diff
@@ -460,6 +460,28 @@ with all the popular application servers -- Unicorn, Puma and Passenger.
 Action Cable does not work with WEBrick, because WEBrick does not support the
 Rack socket hijacking API.
 
+## npm usage
+
+In addition to being available through the `actioncable` gem, Action Cable's frontend JS assets are also bundled in an officially supported npm
+module, intended for usage in standalone frontends that communicate with a Rails
+API.
```

Comment:
Go broader. Discuss asset packaging options. Most apps will use Action Cable with the asset pipeline (…and here's how) but you can use other asset packagers, too (and here's how).